### ISSUE TITLE
Docs side-navigation should be hidden from keyboard users in tablet/mobile view

### ISSUE THREAD

**Issue by marcysutton** [angular/material#234]
As a desktop keyboard user, when my browser window is small enough to trigger the tablet or mobile layouts, `material-sidenav` links are still in the tab order even though they are offscreen. This forces me through a menu I cannot see.  To fix, links that are not visible should have `tabIndex="-1"` or be put inside of an element with CSS of `display: none`.

![Tablet view on desktop](https://cloud.githubusercontent.com/assets/1045233/4112837/1a3bc29e-323b-11e4-9169-4bde670d59cf.png)
